game/src/core/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+project(core)
+
+file(GLOB SOURCES *.cc)
+file(GLOB HEADERS *.h)
+
+include_directories(.)
+
+add_library(core SHARED ${SOURCES} ${HEADERS})
+
+target_link_libraries(core lua util)
+target_link_libraries(core ${LUA_LIBRARIES})
```

Comment:
I'm not sure about adding `LUA_LIBRARIES` here. Please tell me if I should remove this line.